src/core/core.scale.js

```diff
@@ -569,6 +569,60 @@ module.exports = function(Chart) {
 				0;
 		},
 
+		/**
+		 * Returns a subset of ticks to be plotted.
+		 * This functionality makes it so that we don't have overlapping labels.
+		 */
+		autoSkip: function() {
```

Comment:
I would call it `_disseminate` and make it explicitly `@private`. I would also add the `ticks` as parameter